src/status_im/models/transactions.cljs

```diff
@@ -218,7 +219,7 @@
             result))))
 
 (defn- etherscan-transactions [chain account on-success on-error]
-  (if (etherscan-supported? chain)
+  (if (and (etherscan-supported? chain) (not config/chaos-mode?))
     (let [url (get-transaction-url chain account)]
       (log/debug "HTTP GET" url)
       (http/get url
```

Comment:
I would go as far, as tweak this in `http/get` actually, to make it as similar to the real "etherescan is down" scenario as possible.

Reply:
I just replaced urls, is it enough? :)